internal/appconfig/serde.go

```diff
@@ -15,6 +15,12 @@ import (
 	"github.com/superfly/flyctl/iostreams"
 )
 
+const flytomlHeader = `# fly.toml app configuration file generated for %s on %s
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+`
+
```

Comment:
This is *such* a nitpick and it doesn't _really_ matter but IMO it looks nicer when there's a new line between the header comment and the content. This is a really nice change!
```suggestion
const flytomlHeader = `# fly.toml app configuration file generated for %s on %s
#
# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
#

`

```